ui/pages/permissions-connect/flask/util.js

```diff
@@ -17,7 +17,9 @@ export function getSnapInstallWarnings(permissions, targetSubjectMetadata, t) {
   return [
     ...flatMap(bip32EntropyPermissions, (permission, i) =>
       permission.caveats[0].value.map(({ path, curve }) => ({
-        id: `key-access-bip32-${path.join('/')}-${curve}-${i}`,
+        id: `key-access-bip32-${path
+          .join('-')
+          .replace(/\'/g, 'h')}-${curve}-${i}`,
```

Comment:
```suggestion
          .replace(/'/gu, 'h')}-${curve}-${i}`,
```
Escaping is not needed and the linter wants `/gu`